faker/providers/person/__init__.py

```diff
@@ -25,6 +27,10 @@ def first_name(cls):
     def last_name(cls):
         return cls.random_element(cls.last_names)
 
+    @classmethod
+    def middle_name(cls):
```

Comment:
Not every provider supports middle names. You can move this logic to the localized russian subclass (`person/ru_RU/__init__.py`)